test/index.test.js

```diff
@@ -60,13 +55,24 @@ describe('fixtures', () => {
 			await rimraf(resolve(`${fixturePath}/.rts2_cache_es`));
 			await rimraf(resolve(`${fixturePath}/.rts2_cache_umd`));
 
-			let script;
-			try {
-				({ scripts: { build: script } = {} } = JSON.parse(
-					await readFile(resolve(fixturePath, 'package.json'), 'utf8'),
-				));
-			} catch (err) {}
-			script = script || DEFAULT_SCRIPT;
+			async function getBuildScript() {
+				try {
+					const pkgJSON = await readFile(
```

Comment:
actually would be better to check if this file exists before reading it unconditionally - that way lack of package.json could be handled more explicitly here

Reply:
Yes, that's better